src/SayMoreTests/ApplicationContainerTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Linq;
+using NUnit.Framework;
+using SayMore;
+using SayMore.Model.Files;
+using SayMore.Properties;
+
+namespace SayMoreTests
+{
+	[TestFixture]
+	public class ApplicationContainerTests
+	{
+		[Test]
+		public void ComponentRoles_ContainsOralTranslationRole_HasCorrectRenamingSuffix()
+		{
+			// get list of all available roles
+			var roles = ApplicationContainer.ComponentRoles.ToList();
```

Comment:
No need to get these into a list. Also, since we only expect one match, instead of FirstOrDefault, call Single(r => r.Id == ComponentRole.kOralTranslationComponentRoleId) directly on ApplicationContainer.ComponentRoles.

Reply:
Done.